spec/features/admin/forms_spec.rb

```diff
@@ -751,6 +751,19 @@
           end
         end
 
+        describe 'editing form timezone' do
+          before do
+            select(US_TIMEZONES.sample[1], from: 'form_time_zone')
```

Comment:
here's a test to ensure the timezone selected for a form (from the UI) is valid, by checking that the submissions (which use `formatted_time` still load properly).